src/main_controller.ts

```diff
@@ -57,14 +57,14 @@ export class MainController implements vscode.Disposable {
     public commandLineManager!: CommandLineManager;
     public statusLineManager!: StatusLineManager;
     public highlightManager!: HighlightManager;
-    public multilineMessagesManager!: MultilineMessagesManager;
+    public messagesManager!: MessagesManager;
     public viewportManager!: ViewportManager;
 
     public constructor(private extContext: ExtensionContext) {}
 
-    public async init(): Promise<void> {
+    public async init(outputChannel: vscode.LogOutputChannel): Promise<void> {
         const [cmd, args] = this.buildSpawnArgs();
-        logger.debug(`Spawning nvim: ${cmd} ${args.join(" ")}`);
+        logger.info(`starting nvim: ${cmd} ${args.join(" ")}`);
```

Comment:
Log Nvim start-command at startup. Will add more info logs in the future (e.g. the result of `stdpath('config')`)